project/FilteredJacocoAgentPlugin.scala

```diff
@@ -70,17 +70,34 @@ object FilteredJacocoAgentPlugin extends AutoPlugin {
     }
   }
 
+  lazy val Jmf = config("jmf").extend(Compile)
+
   // ---- commands
   private lazy val jacocoCleanAllCmd  = Command.command("jacocoCleanAll") { state =>
     val targets = enabledUnder(state)
     if (targets.isEmpty) { println("[jacoco] nothing to clean (no enabled modules under this aggregate)."); state }
     else targets.foldLeft(state) { (st, ref) => Command.process(s"${ref.project}/jacocoClean", st) }
   }
 
-  private lazy val jacocoReportAllCmd = Command.command("jacocoReportAll") { state =>
-    val targets = enabledUnder(state)
-    if (targets.isEmpty) { println("[jacoco] nothing to report (no enabled modules under this aggregate)."); state }
-    else targets.foldLeft(state) { (st, ref) => Command.process(s"${ref.project}/jacocoReport", st) }
+  private lazy val jacocoReportAllCmd = Command.command("jacocoReportAll") { state0 =>
```

Comment:
```suggestion
  private lazy val jacocoReportAllCmd = Command.command("jacocoReportAll") { state =>
```

Naming suggests that there are multiple states, `state0` preceding another `state`, I but I don't see that here?

Or do you mean that this is an _initial state_?

Reply:
Fixed in commit cf0151f8a16adaf444d05d5189b4514d7cd5b259.

Reply:
I have reverted this as the change introduces this error:  configuration id must be capitalized.

Reply:
Surprising, but ok